Sources/Jaeger/Jaeger/Agent/CoreData/AgentCoreDataObjectModel.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  AgentCoreDataObjectModel.swift
+//  Jaeger
+//
+//  Created by Simon-Pierre Roy on 2/1/19.
+//
+
+import CoreData
+
+/// The Core Data model for the Agent written in Code.
+enum AgentCoreDataObjectModel {
+
+    /// The Core Data model for the Agent written in Code.
+    static let model: NSManagedObjectModel = {
+
+        let model = NSManagedObjectModel()
+
+        // Core Data Span
+        let coreDataSpan = NSEntityDescription()
+        coreDataSpan.name = "CoreDataSpan"
+        coreDataSpan.managedObjectClassName = "CoreDataSpan"
```

Comment:
I don’t know Core Data very well, but might it make sense to put a comment here directing people to the location of where this class is actually defined?

Reply:
Yes. (It is in the same folder). I will add a link to it in the doc :)